### ISSUE TITLE
Using Pocket to generate post objects in WordPress

### ISSUE THREAD

**Issue by mor10** [bamadesigner/pocket-to-wordpress#1]
Some initial thoughts: There is a "light" and a "heavy" way to go with this. The "light" way would be to pull content from the Pocket API dynamically and display the results on a single page. The "heavy" way - which IMO is preferred - is to use the Pocket API as a trigger for the automated creation of new WordPress post objects - most likely in the form of a Link post. That way once something is flagged on Pocket, a persistent, queryable, and independent entry is created that will live on the site the plugin runs on.

On a wider basis this could allow for the creation of custom index templates to display and filter posts based on variables like time and tags.

**Comment by bamadesigner** [comment]
Idea: what if we set it up so you set a specific tag that enabled the automatic post create, e.g. If I want to automatically send a Pocket article to my WordPress site as a post object, I tag it with "autopost", or something a little more attractive. We could also set it up so you just list out the tags you want to automatically post then you can just list out a bunch of your favorite tags. Or you can set it to automatically post everything you pocket.

**Comment by mor10** [comment]
Yes, exactly! And then for kicks we can make the specific tag a user setting so multilingual users can choose whatever tag they want as the trigger.

**Comment by bamadesigner** [comment]
My thoughts exactly!
